scripts/keeper.ts

```diff
@@ -71,24 +70,30 @@ export async function flush(config: Config) {
   );
 
   const domainsToFlush = await getDomainsToFlush(l2TeleportGateway, l1Signer, config);
-  domainsToFlush.forEach(async (targetDomain) => {
+  config.targetDomains.forEach(async (targetDomain: string) => {
     const [daiToFlushSplit] = await l2TeleportGateway.batched_dai_to_flush(
       targetDomain
     );
     const daiToFlush = toUint(daiToFlushSplit);
     console.log(`DAI to flush: ${daiToFlush}`);
 
-    if (daiToFlush > config.flushMinimum) {
+    const sendFlush = async () => {
       console.log("Sending `flush` transaction");
-      const { amount } = await l2TeleportGateway.estimateFee.flush([
-        targetDomain 
-      ]);
+      console.log(` Domain: ${targetDomain}`);
+      console.log(` Amount: ${daiToFlush}`);
+      const { amount } = await l2TeleportGateway.estimateFee.flush(targetDomain);
       const { transaction_hash } = await l2TeleportGateway.flush(
         targetDomain,
         { maxFee: amount * config.l2GasMultiplier }
       );
       await l2Signer.waitForTransaction(transaction_hash);
       console.log("Success");
+    }
+
+    if (daiToFlush > config.flushMinimum) {
+      await sendFlush();
+    } else if (daiToFlush > 0 && domainsToFlush.includes(targetDomain)) {
```

Comment:
it should be: `daiToFlush > 0 && !domainsToFlush.includes(targetDomain)` ? I mean flush if there were no recent flushes.